R/class-yspec.R

```diff
@@ -406,14 +406,21 @@ yspec_yml_file.default <- function(x,...) {
 
 #' Add label attribute to data set columns
 #' 
+#' This is commonly done prior to saving data to SASxport or other formats
+#' that will track label metadata for a data column.
 #' 
-#' @param data a `data.frame` to label
-#' @param spec yspec object for `data`
-#' @param fun the function to use for forming `label`
+#' @param data a `data.frame` to label.
+#' @param spec yspec object for `data`.
+#' @param fun the function to use for forming `label`.
+#' @param strict if `TRUE`, generate an error when `data` names do not match 
+#' the `spec` names; otherwise, only data items in common between `data`
+#' and `spec` will be labeled.
```

Comment:
From reading this, I'm not sure if it'd be obvious to me that "match" means `identical` rather than `setequal` (i.e. that order matters).  May be worth explicitly mentioning order in this description.

Reply:
I clarified the language to include / exclude order. This makes it seem very strict, but the use case for this function is really labeling _the_ data set corresponding to _the_ spec ... formally writing the data set to `.xpt`; but now there's a workaround for the fringe cases where we want to label whatever happens to be there.